src/PGFPlots.jl

```diff
@@ -787,6 +807,19 @@ function axisOptions(p::Image)
     end
 end
 
+function axisOptions(p::MatrixPlot)
+    if p.colorbar
+        cmOpt = colormapOptions(p.colormap)
+        if p.colorbarStyle == nothing
```

Comment:
It'd be easier to understand if this was changed to:
```julia
option_string = "enlargelimits = false, axis on top, $cmOpt, colorbar, xmin=$(p.xmin), xmax=$(p.xmax), ymin=$(p.ymin), ymax=$(p.ymax)"
if p.colorbarStyle == nothing
    option_string *= ", style = {$(p.colorbarStyle)}"
end
return option_string

Reply:
I think it needs to be the other way around, so I adjusted it to this
```julia
option_string = "enlargelimits = false, axis on top, $cmOpt, colorbar, xmin=$(p.xmin), xmax=$(p.xmax), ymin=$(p.ymin), ymax=$(p.ymax)"
if p.colorbarStyle != nothing
    option_string *= ", style = {$(p.colorbarStyle)}"
end
    return option_string
```